### PR TITLE
Add folder level mute config for SCC v2

### PR DESCRIPTION
**Pull request by thokalavinod**
Adding new template for SCC API V2 Folder Mute Config
https://cloud.google.com/security-command-center/docs/reference/rest/v2/folders.muteConfigs

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_scc_v2_folder_mute_config`
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test .go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package securitycenterv2_test
```
Comment (hao-nan-li) [inline mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test .go:1]: There's a trailing white space in this file name.

**Review by hao-nan-li — COMMENTED** [review]
Looks like there's a build error, could you follow the instructions at https://googlecloudplatform.github.io/magic-modules/get-started/generate-providers/ and try to resolve it?

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test .go (hunk 1 above)
Comment (thokalavinod) [reply mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test .go]: addressed

**Comment by thokalavinod** [comment]
[Tf test debug log for folder mute config.txt](https://github.com/user-attachments/files/16433693/Tf.test.debug.log.for.folder.mute.config.txt)
I am attaching the debug log for folder mute config file. Locally tests are passing.

File: mmv1/products/securitycenterv2/FolderMuteConfig.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    name: mute_config_id
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Unique identifier provided by the client within the parent scope.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
```
Comment (hao-nan-li) [inline mmv1/products/securitycenterv2/FolderMuteConfig.yaml:69]: Is this field always an output field?

Comment (thokalavinod) [reply mmv1/products/securitycenterv2/FolderMuteConfig.yaml]: It's not mentioned in the document. I am removing it.

Comment (thokalavinod) [reply mmv1/products/securitycenterv2/FolderMuteConfig.yaml]: Its not mentioned in the doccument. I removed it.

**Review by hao-nan-li — APPROVED** [review]

**Pull request merged**